packs/data/the-slithering-bestiary.db/fuming-sludge.json

```
@@ -27,9 +27,21 @@
                     }
                 },
                 "description": {
-                    "value": "<p>[[/r 1d8[persistent,fire]]] damage</p>"
+                    "value": ""
```

```suggestion
                    "value": "Plus 1d8 persistent fire damage"
```
We should note this somewhere until we can make it show in the actual damage. That way the GM knows it's coming at least.